docs/getting-started/installation.md

```diff
@@ -251,3 +251,18 @@ podAnnotations: {}
 [ecr]: https://gallery.ecr.aws/aquasecurity/trivy
 [registry]: https://github.com/orgs/aquasecurity/packages/container/package/trivy
 [helm]: https://github.com/aquasecurity/trivy/tree/{{ git.tag }}/helm/trivy
+
+
+## pre-commit hook
+
+Add this to your [pre-commit](https://pre-commit.com/) `.pre-commit-config.yaml` config:
+
+```yaml
+repos:
+-   repo: https://github.com/aquasecurity/trivy.git
+    rev: {{ git.tag }}
+    hooks:
+    -   id: trivyfs
```

Comment:
what if we add information about the possibility of using [Trivy flags](https://aquasecurity.github.io/trivy/v0.29.2/docs/references/cli/fs/)?
for example:
```
      args:
        - --severity=CRITICAL
        - --skip-dirs=./test       
```